toplevel/coqinit.ml

```diff
@@ -118,7 +118,7 @@ let init_ocaml_path () =
   let open Loadpath in
   let lp s = { recursive = false; path_spec = MlPath s } in
   let add_subdir dl =
-    Loadpath.add_coq_path (lp (List.fold_left (/) Envars.coqroot [dl]))
+    Loadpath.add_coq_path (lp (List.fold_left (/) (Envars.coqlib()) [dl]))
```

Comment:
This deserves its own commit and a better explanation, please also use more descriptive commit messages.